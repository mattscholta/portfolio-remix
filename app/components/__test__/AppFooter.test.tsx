import { expect, test } from "vitest";
import { render } from "@testing-library/react";

import { AppFooter } from "../AppFooter";
import { MemoryRouter } from "react-router";

describe("AppFooter Component", () => {
  const component = render(
    <MemoryRouter>
      <AppFooter />
    </MemoryRouter>
  );

  test("should render our footer", () => {
    expect(component.baseElement).toMatchSnapshot();
  });
});
