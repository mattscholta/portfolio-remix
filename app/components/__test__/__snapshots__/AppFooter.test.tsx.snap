// Vitest Snapshot v1

exports[`AppFooter Component > should render our footer 1`] = `
<body>
  <div>
    <footer
      class="mt-10 justify-center gap-2 text-center text-sm print:hidden md:mt-20"
    >
      <div
        class="m-auto flex justify-center gap-4"
      >
        <a
          class="p-2"
          href="https://linkedin.com/in/MatthewScholta"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="Follow me on LinkedIn"
            class="footer-social"
            height="16"
            loading="lazy"
            src="/images/svg/linkedin-dark.svg"
            width="16"
          />
        </a>
        <a
          class="p-2"
          href="https://github.com/visormatt"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="Follow me on GitHub"
            class="footer-social"
            height="16"
            loading="lazy"
            src="/images/svg/github-dark.svg"
            width="16"
          />
        </a>
        <a
          class="p-2"
          href="https://twitter.com/visormatt"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="Follow me on Twitter"
            class="footer-social"
            height="16"
            loading="lazy"
            src="/images/svg/twitter-dark.svg"
            width="16"
          />
        </a>
      </div>
      <p
        class="font-font-monospace"
      >
        Built with 
        <span
          class="text-color-primary--"
        >
          â™¥
        </span>
         in
         
        <a
          href="https://www.google.com/search?q=san+diego+weather"
          rel="noreferrer"
          target="_blank"
        >
          San Diego
        </a>
        , CA.
      </p>
    </footer>
  </div>
</body>
`;
